src/satellitePropagator/physics/AccelerationComponents/DragComponent.h

```diff
@@ -5,15 +5,130 @@
 #pragma once
 #include "satellitePropagator/debris/Debris.h"
 #include "satellitePropagator/physics/Constants.h"
+#include <map>
 
 namespace Acceleration::DragComponent {
-namespace {
-}
+/**
+ * Table with values for density in different altitudes.
+ * Units: altitude[km],  density[km/m^3]
+ */
+const std::map<double, double> atmosphericDensityMap {
+    // We only care for altitudes > 150km
+    // { 78., 1.584426874818746e-05 },
+    // { 87.31313131, 5.103100531618111e-06 },
+    // { 96.62626263, 0.2036285852445872e-06 },
+    // { 105.93939394, 0.61505553819552e-07 },
+    // { 115.25252525, 1.8246322614222663e-08 },
+    // { 124.56565657, 8.083351848460097e-09 },
+    // { 133.87878788, 3.328037928047479e-09 },
+    { 143.19191919, 1.382227748398691e-09 },
```

Comment:
Just out of curiosity, did you ever compare if this similar to what we had before?